TestCoins.c

```diff
@@ -0,0 +1,95 @@
+// ------------------------------------------------------------------------------------------------
+// Training ~ A training program for new joiners at Metamation, Batch - July 2024.
+// Copyright (c) Metamation India.
+// Ayisha Sameera,GET.
+// ------------------------------------------------------------------
+// TestCoins.c
+// Program on branch Test 2.2
+// Program that tests Coins.c
+// ------------------------------------------------------------------------------------------------
+
+#include <stdio.h>
+#include <stdbool.h>
+#include <ctype.h>
+#include <string.h>
+#include <stdlib.h>
+#include "Coins.h"
+
+static bool isEqual (Coins a, Coins b) {
+   return a.Ten == b.Ten && a.Five == b.Five && a.Two == b.Two && a.One == b.One;
+}
+
+static bool Choice () {
+   while (1) {

```

Comment:
Please go through and implement changes given to you in previous reviews. 
Changes / practices told to you in reviews are ALWAYS general and not particular to that assignment.